src/ui/common/src/components/integrations/integrationObjectList.tsx

```diff
@@ -54,6 +54,12 @@ const IntegrationObjectList: React.FC<Props> = ({ user, integration }) => {
     );
   }, [selectedObject]);
 
+  if (integration.service === 'Kubernetes' || integration.service === 'Lambda') {
+    return (
+      <></>
```

Comment:
This can be '''return null''' instead so you don't have a empty dom node just hanging out.